.github/workflows/code_checks.yml

```diff
@@ -159,12 +159,13 @@ jobs:
                 uses: shivammathur/setup-php@v1
                 with:
                     php-version: 7.3
-                    coverage: none # disable xdebug, pcov
+                    coverage: pcov
 
             -   run: composer install --no-progress
 
+            # https://kizu514.com/blog/pcov-is-better-than-phpdbg-and-xdebug-for-code-coverage/
             -   run: |
-                    phpdbg -qrr -d memory_limit=-1 vendor/bin/phpunit --coverage-clover coverage.xml
+                    vendor/bin/phpunit --coverage-clover coverage.xml
```

Comment:
Based on @krakjoe feedback, we'll give it a try
https://www.reddit.com/r/PHP/comments/eppx8z/how_we_speedup_coverage_on_travis_by_95/fepe9yz?utm_source=share&utm_medium=web2x